xmppserver/src/main/java/org/jivesoftware/openfire/session/Session.java

```diff
@@ -47,7 +48,7 @@
     int STATUS_CLOSED = -1;
     int STATUS_CONNECTED = 1;
     int STATUS_AUTHENTICATED = 3;
-
+    
```

Comment:
Try to avoid having white-space changes in a commit. There's a couple of these in this PR.